app/graphql/mutations/delete_member.rb

```diff
@@ -0,0 +1,18 @@
+module Mutations
+  class DeleteMember < GraphQL::Function
+    argument :id, !types.Int, 'ID of the member getting updated'
+
+    description 'Update a member status'
+    type ::Types::UserType
+
+    def call(_, args)
+      @user = User.find args[:id]
```

Comment:
With my latest branches now merged in, we should now add Pundit authorization here.

Reply:
Yeah I was wondering how that would work in this situation where I have to pick which value to save based on the participation. I'll read more about Pundit to see it.